samples/web-terminal-custom-tooling.yaml

```diff
@@ -24,3 +24,8 @@ spec:
           env:
             - name: PS1
               value: \[\e[34m\]>\[\e[m\]\[\e[33m\]>\[\e[m\]
+          volumeMounts:
+            - path: "/home/"
```

Comment:
Shouldn't we mount to `/home/user`?

Reply:
Changed to /home/user.

but actually, it's strange that I got permissions denied error when I try /home, somebody creates non-writable /home/user.